client/components/BarVersion/BarVersion.scss

```diff
@@ -0,0 +1,46 @@
+@import '../../../stylesheets/colors';
+@import '../../../stylesheets/variables';
+
+.bar-graph__bar-version {
+  @include font-size-xs;
+  z-index: 33;
+  font-weight: $font-weight-light;
+  transform: rotate(-180deg);
+  font-variant-numeric: tabular-nums;
+  color: lighten($raven, 15%);
+  transition: opacity 0.2s, color 0.2s;
+  font-family: $font-family-code;
+  letter-spacing: -1px;
+  line-height: 1;
+  cursor: pointer;
+  height: $bar-width;
+  writing-mode: vertical-rl;
+  text-orientation: sideways;
+  height: 50px;
```

Comment:
There's already a height attribute above (on that note, we should probably add stylelint as well to this project).